services/system/Accounts/ui/package.json

```diff
@@ -51,6 +50,6 @@
     "tailwindcss": "^3.4.10",
     "typescript": "^5.5.3",
     "typescript-eslint": "^8.0.1",
-    "vite": "^5.4.1"
+    "vite": "4.5.2"
```

Comment:
You have 
```
 "devDependencies": {
    "vite": "5.4.1"
  }
```
In `services/package.json`, but then here you install `4.5.2`. Intentional?